docs/.vuepress/public/presets/toml/tokyo-night.toml

```diff
@@ -16,6 +20,52 @@ $time\
 [ ](fg:#1d2230)\
 \n$character"""
 
+# Disable the blank line at the start of the prompt
+add_newline = true
+
+# You can also replace your username with a neat symbol like   or disable this
+[username]
+show_always = true
+style_user = "bg:#a3aed2 fg:#090c0c"
+style_root = "bg:#a3aed2 fg:#090c0c"
+format = '[$user ]($style)'
+disabled = false
```

Comment:
In my perspective, I would prefer it, if the new default, respects the one from the original author.
Meaning, please use `disable = true` for the `[username]` module.

Reply:
Fixed.